### PR TITLE
Offhand fixes

### PR DESCRIPTION
**Pull request by GurteX**
Add your description here...

### CODE REVIEW

**Comment by 3arthqu4ke** [comment]
Ok, will see, didnt really need these features for the bot

File: src/main/java/me/earth/phobot/modules/combat/AutoTotem.java [hunk 1]
```diff
@@ -5,19 +5,30 @@
...
 import net.minecraft.client.multiplayer.ClientLevel;
 import net.minecraft.client.multiplayer.MultiPlayerGameMode;
 import net.minecraft.client.player.LocalPlayer;
+import net.minecraft.world.item.AxeItem;
+import net.minecraft.world.item.Item;
 import net.minecraft.world.item.Items;
+import net.minecraft.world.item.SwordItem;
 
 public class AutoTotem extends PhobotModule {
+    private final Setting<Offhand> mode = constant("Mode", Offhand.None, "What item you prefer in your offhand.");
```
Comment (3arthqu4ke) [inline src/main/java/me/earth/phobot/modules/combat/AutoTotem.java:23]: Hmmm, dont like that this is a setting. I would rather like a system with a bind to switch to a certain mode. I might make that myself at some point. Ok lets be real never, Cause AutoCrystal already switches to crystals for me.

**Comment by GurteX** [comment]
Great! Can zombiebob35 crayons be added tho? 
image

**Review by 3arthqu4ke — CHANGES_REQUESTED** [review]
Would merge Sword-Gap right away, but idk about the mode setting.

File: src/main/java/me/earth/phobot/modules/combat/AutoTotem.java (hunk 1 above)
Comment (GurteX) [reply src/main/java/me/earth/phobot/modules/combat/AutoTotem.java:23]: > Hmmm, dont like that this is a setting. I would rather like a system with a bind to switch to a certain mode. I might make that myself at some point. Ok lets be real never, Cause AutoCrystal already switches to crystals for me.

but what if you arent using phobot for ca, its just an optional setting, None -> It stays the same, as it is rn, but with sword gap and 2 others are just to make it always be totem or crystal

**Review by 3arthqu4ke — APPROVED** [review]

**Pull request merged**